credoai/lens/lens_validation.py

```diff
@@ -125,8 +125,8 @@ def check_prediction_model_output(fn, data, batch: int = 1):
     batch_size = batch
```

Comment:
Why the reassigning of batch?

Reply:
I wanted to avoid confusion between input `batch` and output `batch_size` since its value could be changed depending on the data type of `X`.

This is a primarily internal function so I'm not sure it matters. I can change to batch_out and batch_in.